docs/pages/guides/using-nextjs.mdx

```diff
@@ -6,123 +6,222 @@ description: A guide for integrating Next.js with Expo for the web.
 
 import { Collapsible } from '~/ui/components/Collapsible';
```

Comment:
```suggestion
import { Collapsible } from '~/ui/components/Collapsible';
import { Terminal } from '~/ui/components/Snippet';
```

Adding the statement to import `<Terminal>` component since its used in the document below.